node/lib/package/LocalPackageSource.ts

```diff
@@ -303,6 +305,7 @@ export class LocalPackageSource extends PackageSource {
                 let onboardingPackageName: string =
                         LocalPackageSource.derivePackageName(onboardingId, "onboarding");
                 onboardingModulePath = onboardingPackageName + "/" + onboardingId;
+
```

Comment:
ok. sure. so this should be  below ? Fixed to that now.
onboarding: 'opent2t-onboarding-org-opent2t-test-onboarding/org.opent2t.test.onboarding/thingOnboarding',

---
In reply to: [75431448](https://github.com/openT2T/opent2t/pull/22#discussion_r75431448) [](ancestors = 75431448)